.coveragerc

```diff
@@ -7,6 +7,8 @@ omit =
     urllib3/contrib/appengine.py
     urllib3/contrib/ntlmpool.py
     urllib3/contrib/pyopenssl.py
+    urllib3/contrib/securetransport.py
```

Comment:
I know all the contribs are skipped but it makes me uncomfortable to not have coverage data on this...